### PR TITLE
adding prettier, linting all the code, adding notifications reducer

### LINKED ISSUE #22: refactor `import { openSnackbar } from '../../components/notification/SnackBar';`
gonna put this on the Redux, it's not best practise to call functions of a component like that

### PR DESCRIPTION
**Pull request by silviopaganini**
massive commit as the code is linted properly now =) 

also important to note, I've added the notifications to the reducer so it's fixed #22

### CODE REVIEW

**Comment by andrevenancio** [comment]
oh so semicolons `;` are now invalid in the linter? I know there are some that argue that they are optional. However Automatic Semicolon Insertion is part of the Javascript spec. so lets agree to "kind of optional". (Don't want to open pandora's box on javascript semantics).

I personally think they help avoid accidents when typing / transpiling / compiling

```javascript
var foo = 1
var bar = 2
``` 
can easily turn into
```javascript
var foo = 1var bar = 2
```

which will return (in this specific case) invalid or unexpected token, or can also return unexpected identifier.

But who cares..... MERGE :)

**Comment by tevla** [comment]
it's a good point, but I think these rules are mainly for prettier, that shouldn't throw any linting error, or am I wrong?

**Comment by silviopaganini** [comment]
@andrevenancio never had that sort of problem on modern compilers (i.e. CRA) happy to be wrong but it's just an opinionated lint format

File: src/pages/members/Members.jsx [hunk 1]
```diff
@@ -136,93 +131,107 @@ class Members extends React.Component {
...
+      content = <p>loading...</p>
     } else if (error !== null) {
-      content = (<p>{error}</p>);
+      content = <p>{error}</p>
     } else if (members.length) {
       content = (
         <div className={classes.root}>
           <Grid container spacing={24}>
             {members
               .sort((a, b) => {
```
Comment (grudelsud) [inline src/pages/members/Members.jsx]: sort cards recovered from merge conflict and linted

**Pull request merged**